src/SprykerEco/Zed/ArvatoRss/Persistence/ArvatoRssPersistenceFactory.php

```diff
@@ -8,20 +8,28 @@
 namespace SprykerEco\Zed\ArvatoRss\Persistence;
 
 use Orm\Zed\ArvatoRss\Persistence\SpyArvatoRssApiCallLogQuery;
-use Propel\Runtime\ActiveQuery\Criteria;
 use Spryker\Zed\Kernel\Persistence\AbstractPersistenceFactory;
+use SprykerEco\Zed\ArvatoRss\Persistence\Mapper\ArvatoRssPersistenceMapper;
+use SprykerEco\Zed\ArvatoRss\Persistence\Mapper\ArvatoRssPersistenceMapperInterface;
 
 /**
  * @method \SprykerEco\Zed\ArvatoRss\ArvatoRssConfig getConfig()
- * @method \SprykerEco\Zed\ArvatoRss\Persistence\ArvatoRssQueryContainerInterface getQueryContainer()
  */
 class ArvatoRssPersistenceFactory extends AbstractPersistenceFactory
 {
     /**
      * @return \Orm\Zed\ArvatoRss\Persistence\SpyArvatoRssApiCallLogQuery
      */
-    public function createArvatoRssApiCallLogQuery()
+    public function createArvatoRssApiCallLogQuery(): SpyArvatoRssApiCallLogQuery
     {
-        return SpyArvatoRssApiCallLogQuery::create()->orderByUpdatedAt(Criteria::DESC);
+        return SpyArvatoRssApiCallLogQuery::create();
+    }
+
+    /**
+     * @return \SprykerEco\Zed\ArvatoRss\Persistence\Mapper\ArvatoRssPersistenceMapperInterface
+     */
+    public function createArvatoRssPersistenceMapper(): ArvatoRssPersistenceMapperInterface
```

Comment:
Persistence mappers do not require interfaces.

Reply:
Where I can found info about it?

Reply:
That was @PhilinTv recommendations/suggestions.